src/main/features/dashboard/views/macro/claimStatus/statesPaidRejected.njk

```diff
@@ -1,25 +1,39 @@
+{% from "externalLink.njk" import externalLink %}
 {% from "internalLink.njk" import internalLink %}
-{% from "./utilityMacros.njk" import downloadResponseLink %}
+{% from "./utilityMacros.njk" import downloadResponseLink, downloadTheirResponseLink %}
 
-{% macro statesPaidInFullRejectedClaimantDashboard() %}
+{% macro statesPaidRejectedClaimantDashboard() %}
   {{ t('You’ve rejected the defendant’s admission.') }}
 {% endmacro %}
 
-{% macro statesPaidInFullRejectedClaimantClaimDetails(claim) %}
-  <h2 class="heading-small">{{ t('You’ve rejected the defendant’s admission') }}</h2>
+{% macro statesPaidRejectedClaimantClaimDetails(claim) %}
+  {% if claim.claimantResponse.freeMediation and claim.claimantResponse.freeMediation === YesNoOption.YES.option %}
+    {{ statesPaidRejectedMediationYESClaimantClaimDetails(claim) }}
+  {% else %}
+    {{ statesPaidRejectedMediationNOClaimantClaimDetails(claim) }}
+  {% endif %}
+
+  <p>{{ downloadTheirResponseLink(claim) }}</p>
+{% endmacro %}
+
+{% macro statesPaidRejectedMediationYESClaimantClaimDetails(claim) %}
   {% if claim.response.responseType === 'PART_ADMISSION' %}
     {% set amount = claim.response.amount %}
   {% else %}
     {% set amount = claim.response.paymentDeclaration.paidAmount  %}
   {% endif %}
-  <p>{{ t('They said they owe {{ amount }}.', { amount: amount | numeral }) }}</p>
 
-  {{ _linkToDirectionsQuestionnairePage(claim.externalId) }}
-  <p>{{ t('Your claim won’t proceed if you don’t complete and return the form before 4pm on {{ dqDeadline }}.', {
-      dqDeadline: claim.directionsQuestionnaireDeadline | date
-    }) }}</p>
+  <h2 class="heading-small">{{ t('You’ve rejected the defendant’s admission') }}</h2>
+  <p>{{ t('They said they owe {{ amount }}.', { amount: amount | numeral }) }}</p>
+  <p>{{ t('You’ve both agreed to try mediation. The Small Claims Mediation Service will contact you to arrange an appointment.') }}</p>
+  {{ externalLink('Find out how mediation works', 'https://www.gov.uk/government/publications/small-claims-mediation-service-ex730') }}
+{% endmacro %}
 
-  <p>{{ _downloadResponseLink(claim, 'Download their response') }} {{ t('(PDF)') }}</p>
+{% macro statesPaidRejectedMediationNOClaimantClaimDetails(claim) %}
+  <h2 class="heading-small">{{ t('Wait for the court to review the case') }}</h2>
+  <p>{{ t('You’ve rejected {{ defendantName }}’s response and said you want to take the case to court.',
+      { defendantName: claim.claimData.defendant.name }) }}</p>
+  <p>{{ t('The court will review the case. We’ll email you if we set a hearing date to tell you how to prepare.') }}</p>
```

Comment:
this should be the paper DQs version of this status. this one assumes that DQs have been turned on

Reply:
DQs content is being reviewed by Steve and I've been told to progress as is for now